dbtmetabase/metabase.py

```diff
@@ -859,12 +868,12 @@ def api(
             try:
                 response.raise_for_status()
             except requests.exceptions.HTTPError:
-                if "password" in kwargs["json"]:
+                if "json" in kwargs and "password" in kwargs["json"]:
```

Comment:
At least for the error I was getting with exposures, kwargs does not have "json", it is just the auth token for metabase.

Reply:
@kochalex  Is this related to the same issue as the ticket?

Reply:
It is - the error handling here errors out when dbt-metabase makes a GET for a deleted user when exporting exposures.

That said, I did not dig to see if this was specific to GET "/api/user/deleted_user_id" or a more general case for other API endpoints - I intended for this change to "do no harm"